frontend_v2/src/app/components/profile/profile.component.html

```diff
@@ -133,7 +134,7 @@
                     <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
                   </button>
                 </div>
-                <ul class="inline-list">
+                <ul id="auth_token_buttons" class="inline-list">
```

Comment:
@ShauryaAg it would be good to make this a css class

Reply:
I have changed this to a class.